pkg/daemon/discover/discover.go

```diff
@@ -198,7 +194,7 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 	// get discoverDaemonUdevBlacklist from the environment variable
 	// if user doesn't provide any regex; generate the default regex
 	// else use the regex provided by user
-	discoverUdev := os.Getenv(discoverDaemonUdev)
+	discoverUdev := os.Getenv(k8sutil.DiscoverDaemonUdevBlacklistEnvName)
```

Comment:
This Env Var is only used here and has no use outside this scope.
So, I prefer the older way than exporting to a utility file.

Reply:
It's also used in `pkg/operator/k8sutil/pod.go`, @leseb asked for this change here: https://github.com/rook/rook/pull/5198#discussion_r404680745